src/styles/blocks/clock.scss

```
@@ -0,0 +1,54 @@
+.clock {
+  @include round(500px, 50%);
+  background-color: aquamarine;
+  @extend %shadows;
+
+  &__numbers {
+    position: absolute;
+    top: calc(50% - 125px);
+    left: calc(50% - 125px);
+  }
+}
+
+.clock-minute {
+  position: absolute;
+  top: calc(50% - 180px);
+  left: calc(50% - 6px);
+
+  height: 180px;
+  width: 12px;
+  border-radius: 12px;
+
+  background-color: $color-minute;
+  @extend %shadows;
+
+  transform-origin: center 180px;
+  animation: move 600s steps(10, end) 1s 1 forwards;
+}
```

please use the power of nesting
```suggestion
.clock {
  @include round(500px, 50%);
  background-color: aquamarine;
  @extend %shadows;

  &__numbers {
    position: absolute;
    top: calc(50% - 125px);
    left: calc(50% - 125px);
  }

  &__minute {
    position: absolute;
    top: calc(50% - 180px);
    left: calc(50% - 6px);

    height: 180px;
    width: 12px;
    border-radius: 12px;

    background-color: $color-minute;
    @extend %shadows;

    transform-origin: center 180px;
    animation: move 600s steps(10, end) 1s 1 forwards;
  }
```

---

check all places